controllers/weightsandbiases_controller.go

```diff
@@ -114,23 +115,29 @@ func (r *WeightsAndBiasesReconciler) Reconcile(ctx context.Context, req ctrl.Req
 
 	license := utils.GetLicense(currentActiveSpec, crdSpec, userInputSpec)
 	log.Info("License", "license", license)
-	deployerSpec, err := deployer.GetSpec(license, currentActiveSpec)
-	if err != nil {
-		log.Info("Failed to get spec from deployer", "error", err)
-		// This scenario may occur if the user disables networking, or if the deployer
-		// is not operational, and a version has been deployed successfully. Rather than
-		// reverting to the container defaults, we've stored the most recent successful
-		// deployer release in the cache
-		if deployerSpec, err = specManager.Get("latest-cached-release"); err != nil {
-			log.Error(err, "No cached release found for deployer spec", err, "error")
+
+	// do not ping deployer if airgapped
+	airgapped := os.Getenv("AIRGAPPED") == "true"
```

Comment:
```suggestion
	isAirgapped := strings.EqualFold(os.Getenv("IS_AIRGAPPED"), "true")
```

Use EqualFold

**nit:** name is indicative of what it does. Boolean props should be named using auxiliary verbs such as does, has, is and should. For example, Button uses isDisabled, isLoading, etc.